onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -799,6 +801,11 @@ def _where_input_wrangler(
         reason="fixme: current implementation gets shape inference error",
         test_class_name="TestOutputConsistencyFullGraph",
     ),
+    xfail(
```

Comment:
Is this a bug?

Reply:
It is. A bug I haven't figured out yet

Reply:
> I wonder if there is a way that exporter can also use extra_Opinfo

What's the scenario of using these op information during exporting?

Reply:
We can probably enrich the pytorch opinfo db @titaiwangms